indexer/go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/ethereum-optimism/optimism/indexer
 
 go 1.17
 
-replace github.com/ethereum/go-ethereum v1.10.26 => github.com/ethereum-optimism/op-geth v0.0.0-20230214215134-401b7fd3309b
+replace (
+	github.com/ethereum/go-ethereum v1.10.26 => github.com/ethereum-optimism/op-geth v0.0.0-20230214215134-401b7fd3309b
+	github.com/ethereum/go-ethereum v1.11.2 => github.com/ethereum-optimism/op-geth v1.11.2-aea0402.0.20230227230209-0705cf1b7df9
```

Comment:
We shouldn't have to specify this replace of geth v1.11.2 if we still use geth v1.10.26 as dependency

Reply:
I'll open a PR to update it, doesn't have to block this